doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,28 @@
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+# InfiniTime
+
+InfiniTime is a community built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
+
+InfiniTime is not to be used for medical or other health tracking purposes.
+
+# Core Principles
+
+- KISS
+- Reliability
+- Battery efficiency
+- Easy and simple navigation
+- Behaviour should be predictable and easy to understand
+- Prefer solid default experience over customization
+- Personalization is achieved through custom watchfaces. More options may be available through a companion app.
```

Comment:
for markdown files it is a good custom to have one line per sentence. Then diffs are easier to read and one word change will only change one line, instead of possible a whole paragraph

```suggestion
- Personalization is achieved through custom watchfaces.
  More options may be available through a companion app.
```